src/components/FlowRunsAccordion.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <template v-if="flowIds">
+    <p-accordion :sections="flowIds" class="flow-runs-accordion">
+      <template #header="{ section: flowId, id, toggle, content, selected }">
+        <FlowRunsAccordionHeader :flow="getFlow(flowId)!" v-bind="{ id, content, toggle, filter, selected }" />
```

Comment:
I think the need for a non-null assertion is a smell test that the flow lookup isn't quite correct - it doesn't seem like an in-memory lookup based on a single API request is likely to handle scale gracefully if the number of flows exceeds the default API limit

Reply:
I might be misunderstanding what's going on here though

Reply:
Came up with a better solution for the type issue. As for the api scale, noted but if this component is trying to show more than the api limit this would be broken from a design perspective and we'd have to rethink that. So for now I'm going to leave this as is.